mediator/esmFldsExchange_cesm_mod.F90

```diff
@@ -96,16 +96,16 @@ subroutine esmFldsExchange_cesm(gcomp, phase, rc)
     type(InternalState) :: is_local
     integer             :: n, ns
     character(len=CL)   :: cvalue
-    logical             :: wavice_coupling
+    logical             :: wav_coupling_to_cice
     logical             :: ocn2glc_coupling
     character(len=*) , parameter   :: subname=' (esmFldsExchange_cesm) '
     !--------------------------------------
 
     rc = ESMF_SUCCESS
 
-    call NUOPC_CompAttributeGet(gcomp, name='wavice_coupling', value=cvalue, rc=rc)
+    call NUOPC_CompAttributeGet(gcomp, name='wav_coupling_to_cice', value=cvalue, rc=rc)
```

Comment:
Is this the feature not turned on in UFS WM yet? My understanding is that currently we only have cice to wave coupling in the UFS coupled runs.

Reply:
This is the cesm exchange fields file and it is a correction to the name of the scheme to match what we used on the WW3 side. 

We don't actually have any test case with cice-wave coupling in UFS but I think we should add one.